src/core/server/metrics/collectors/server.ts

```diff
@@ -26,12 +26,12 @@ interface ServerResponseTime {
 }
 
 export class ServerMetricsCollector implements MetricsCollector<OpsServerMetrics> {
-  private readonly requests: OpsServerMetrics['requests'] = {
+  private requests: OpsServerMetrics['requests'] = {
     disconnects: 0,
     total: 0,
     statusCodes: {},
   };
```

Comment:
dont want optional properties, and as calling `reset` in the constructor is not sufficient to have TS consider the variable as initialized, I'm forced to duplicate that in the `reset` function.